app/assets/javascripts/components/portfolio/_portfolio_form.js.jsx

```diff
@@ -89,8 +109,13 @@ var PortfolioForm = React.createClass({
         <div className="container">
           {
             allProps.portfolio == null
-            ? <button className="btn waves-effect waves-light portfolio-btns" type="submit" name="action" value="Save Profile" onClick={this.handleInsert}>Save Profile</button>
-            : <button className="btn waves-effect waves-light portfolio-btns" type="submit" name="action" value="Save Profile" onClick={this.handleUpdate}>Save Profile</button>
+            ?
+            <div>
+              <button className="btn waves-effect waves-light portfolio-btns" type="submit" name="action" value="Save Draft" onClick={this.handleDraftInsert}>Save as Draft</button>
+              <button className="btn waves-effect waves-light portfolio-btns" type="submit" name="action" value="Save Profile" onClick={this.handleInsert}>Save Profile</button>
+            </div>
+            :
+            <button className="btn waves-effect waves-light portfolio-btns" type="submit" name="action" value="Save Profile" onClick={this.handleUpdate}>Save Profile</button>
           }
```

Comment:
@neight-allen I added the requested change to this branch.

FYI @drod1000 @mollybrown @jdconrad89